frontend/src/stores/gameStore.ts

```diff
@@ -36,14 +49,32 @@ export const useGameStore = defineStore('games', {
 			await this.refreshAllGames();
 		},
 
-		isAvailable(): boolean {
-			const me: User = useUserStore().getMe;
-			// if (!me.isOnline)
-			// retunr false;
-			me.games.forEach((game: Game) => {
-				if (game.state === gameStates.ACTIVE) return false;
-			});
-			return true;
-		},
+		// isAvailable(): boolean {
+		// 	const me: User = useUserStore().getMe;
+		// 	if (!me.isOnline)
+		// 	   return false;
+		// 	me.games.forEach((game: Game) => {
+		// 		if (game.state === gameStates.ACTIVE) return false;
+		// 	});
+		// 	return true;
+		// },
```

Comment:
are all the comments for the next step, or is it just old code? Because then it can probably be removed?

Reply:
this is not mine, but it did crash my code so I didn't wanted to delete it..